src/Twitter.php

```diff
@@ -46,19 +46,28 @@ public static function optionsWidget(array &$element, array $options) {
 
   /**
    * Returns a link field containing a link to `http://twitter.com/share`.
+   *
+   * @return array|null
```

Comment:
Before the change this always returned a renederable array. It should still do so after the changes.